docs/reference/ccr/apis/follow/get-follow-stats.asciidoc

```diff
@@ -179,6 +179,12 @@ Continuing with the fields from `shards`:
   task; in this situation, the following task must be resumed manually with the
   <<ccr-post-resume-follow,resume follower API>>
 
+==== Authorization
+
+If the {es} {security-features} are enabled, you must have `read` and `monitor`
```

Comment:
Cluster monitor privileges are needed to use this api in the follower cluster.

Reply:
Ditto, I'm not certain if it also needs cluster monitor privileges or only needs them.